app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -1,98 +1,58 @@
-import React, { useState, useEffect } from "react";
+import React, { useState } from "react";
 
-import EmptyNotesListImage from "images/EmptyNotesList";
-import { Button, PageLoader } from "neetoui/v2";
-import { Container, Header, SubHeader } from "neetoui/v2/layouts";
-
-import notesApi from "apis/notes";
-import EmptyState from "components/Common/EmptyState";
+import { Button } from "neetoui/v2";
+import { Header } from "neetoui/v2/layouts";
 
+import { SEARCH_PLACEHOLDER } from "./constants";
 import DeleteAlert from "./DeleteAlert";
-import NoteTable from "./NoteTable";
+import NoteList from "./List";
 import NewNotePane from "./Pane/CreateNote";
 
+import Menubar from "../../Common/Menubar";
```

Comment:
It's better to create an alias for `Common` since it will be used at multiple places.